sw/tools/rtp_viewer/rtp_viewer.py

```diff
@@ -72,7 +78,7 @@ def on_key(self, key):
             self.mouse['start'] = None
 
     def on_mouse(self, event, x, y, flags, param):
-        if event == cv2.EVENT_LBUTTONDOWN and self.rotate == 0:
+        if event == cv2.EVENT_LBUTTONDOWN and self.rotate == 0 and False:
```

Comment:
Defo a different PR. @wilcoschoneveld  can you look at this? I think I will turn it back on for now and make an issue for this.